test_conformance/conversions/test_conversions.cpp

```diff
@@ -340,82 +391,67 @@ int main (int argc, const char **argv )
 #pragma mark -
 #pragma mark setup
 
-static int ParseArgs( int argc, const char **argv )
+static int ParseArgs(int argc, const char **argv)
 {
     int i;
-    argList = (const char **)calloc( argc - 1, sizeof( char*) );
+    argList = (const char **)calloc(argc, sizeof(char *));
```

Comment:
Are we missing the -1 here?

Reply:
I don't know, but I think this was needed because otherwise GCC will think that the array size could be zero, in which case it would warn on accessing `argList`. I remember encountering something similar in `test_compiler_defines_for_extensions.cpp`, which was fixed upstream in a similar fashion.